models/brac/impact/iccm/get_dashboard_data_iccm_impact_u1.sql

```diff
@@ -202,23 +203,23 @@ SELECT
 	SUM(COALESCE(assess_count.count_ax_with_fu_tx_complete,0)) AS count_ax_with_fu_tx_complete,
 	SUM(COALESCE(assess_count.count_ax_with_fu_tx_given_during_ax,0)) AS count_ax_with_fu_tx_given_during_ax,
 	
-	SAFE_DIVIDE(SUM(COALESCE(assess_count.count_ax_with_fu_complete,0)),SUM(COALESCE(assess_count.count_ax_with_fu_rec,0)),2) AS percent_ax_with_fu_complete,
-	SAFE_DIVIDE(SUM(COALESCE(assess_count.count_ax_with_fu_complete_on_time,0)),SUM(COALESCE(assess_count.count_ax_with_fu_rec,0)),2) AS percent_ax_with_fu_complete_on_time,
-	SAFE_DIVIDE(SUM(COALESCE(assess_count.count_ax_with_fu_complete_patient_improved,0)),SUM(COALESCE(assess_count.count_ax_with_fu_complete_patient_status_checked,0)),2) AS percent_ax_with_fu_complete_patient_improved,
-	SAFE_DIVIDE(SUM(COALESCE(assess_count.count_ax_with_fu_ref_complete,0)),SUM(COALESCE(assess_count.count_ax_with_fu_ref_rec,0)),2) AS percent_ax_with_fu_ref_complete,
-	SAFE_DIVIDE(SUM(COALESCE(assess_count.count_ax_with_fu_ref_complete_attend_hf,0)),SUM(COALESCE(assess_count.count_ax_with_fu_ref_complete,0)),2) AS percent_ax_with_fu_ref_complete_attend_hf,
-	SAFE_DIVIDE(SUM(COALESCE(assess_count.count_ax_with_fu_tx_complete,0)),SUM(COALESCE(assess_count.count_ax_with_fu_tx_rec,0)),2) AS percent_ax_with_fu_tx_complete,
-	SAFE_DIVIDE(SUM(COALESCE(assess_count.count_ax_with_fu_tx_given_during_ax,0)),SUM(COALESCE(assess_count.count_ax_with_immediate_tx_rec,0)),2) AS percent_ax_with_fu_tx_given_during_ax,	
+	{{ ref("SAFE_DIVIDE") }} (SUM(COALESCE(assess_count.count_ax_with_fu_complete,0)),SUM(COALESCE(assess_count.count_ax_with_fu_rec,0)),2) AS percent_ax_with_fu_complete,
```

Comment:
ditto on safe_divide naming

Reply:
IMHO, this syntax makes it harder to read. Unsure if there are better alternatives

Reply:
No better alternatives, no.

It is the same as the old syntax only moved to the left instead of inclusive brackets.